app/assets/javascripts/modules/namespaces/components/visibility.js

```diff
@@ -51,6 +51,7 @@ export default {
 
     change(visibility) {
       const currentVisibility = this.namespace.visibility;
+      var params;
```

Comment:
In ES6 we don't use `var` anymore but only `let` or `const`. If you are going to be reassigning the value `let` is your choice, otherwise use `const`. In this case we can use `const` but directly on the line 63.